pilot/pkg/networking/util/util.go

```diff
@@ -212,11 +209,6 @@ func BuildAdditionalAddresses(extrAddresses []string, listenPort uint32, node *m
 	return additionalAddresses
 }
 
-// BuildAddress returns a SocketAddress with the given ip and port or uds.
-func BuildInternalAddress(name string) *core.Address {
```

Comment:
May be also rename BuildInternalAddressWithIdentifier -> BuildInternalAddress?

Reply:
ok